### ISSUE TITLE
http method

### ISSUE THREAD

**Issue by lp-wodell** [LivePersonInc/developers-community#2]
The EAs API says you need to use POST with x-http-override of PUT, but I found it only works if you use PUT directly?

**Comment by EdenKupe** [comment]
Forwarded to Product Manager, answer soon!

**Comment by lp-wodell** [comment]
@EdenKupe: A client is about to use this API themselves and because I am away in training this week I have pointed them at this URL, so quick clarification if there are any issues would be great.

**Comment by EdenKupe** [comment]
Hey,

Got a response from the relevant team. It seems like the method name you mentioned above it lacking. You used " x-http-override" while the name of the header in the documentation is "x-http-method-override" (you left out "method"). 

Using the correct header name exactly as specified in the documentation will yield the required result. 
Having said that, calling the API using the PUT method "directly" (as you mentioned) should also work, and will also yield the same result. 

Bottom line, calling this API will work OK in both ways:
1. method POST + header "x-http-method-override" with the "PUT" value
2. method PUT 

Hope this answers your question! Let me know if not and I'll re-open.

**Comment by lp-wodell** [comment]
Sorry Eden, it doesn't, because I just mis-typed the header in my original post. 

I am actually correctly using 'x-http-method-overrride' and it's not working unless I use the PUT method.


Example:
```
POST /api/account/52560147/monitoring/visitors/{{sharkVisitorId}}/visits/current/events?v=1&amp;sid={{sharkSessionId}} HTTP/1.1
Host: lo.v.liveperson.net
Content-Type: application/json
x-http-method-overrride: PUT
Authorization: OAuth oauth_consumer_key="5e60552c5ae34c5fa64e709edb737ff0",oauth_token="063a15e59fd94b1b974ae33ef9109401",oauth_signature_method="HMAC-SHA1",oauth_timestamp="1498493687",oauth_nonce="dcZuQz7yNuz",oauth_version="1.0",oauth_signature="pHiuBBcdd8TZYfaK%2FphT7lNWX1M%3D"
Cache-Control: no-cache
Postman-Token: 032e4e76-85c4-a537-8bd5-e6ab618b3eba

    [{
			"type": "ctmrinfo",
			"info": {
				"customerId": "RBS-123-456-789-010"
			}
	}]
```
